### PR TITLE
Feature/num 507 integration test chronic lung diseases

### PR DESCRIPTION
**Pull request by af-dpe**
Add 4 testcases
add 4 jsons
* chronic lung diseases
* chronic obstructive lung disease
* ~~infectious disease of lung - disorder~~
* fibrosis of lung

### CODE REVIEW

File: src/test/resources/Condition/create-chronic-lung-disease-infectious-disease-disorder.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "id": "77a8826d-4397-4937-8cca-de90008d3368",
+    "meta": {
+      "profile": [
+        "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/complications-covid-19"
```
Comment (wlad) [inline src/test/resources/Condition/create-chronic-lung-disease-infectious-disease-disorder.json]: @af-dpe  are you sure this is the correct example?

Comment (wlad) [reply src/test/resources/Condition/create-chronic-lung-disease-infectious-disease-disorder.json]: as agreed please remove TC and related example JSON

File: tests/robot/CONDITION/01_create.robot [hunk 2]
```diff
@@ -153,3 +155,51 @@ Force Tags              create    condition
...
+    ...                3. post example json to observation endpoint
+	...                4. validate the response status
+    [Tags]             chronic-lung-diseases    valid    not-ready
+
+    ehr.create new ehr    000_ehr_status.json
+    condition.create chronic lung diseases    Chronic obstructive lung disease    create-chronic-obstructive-lung-disease.json
+    condition.validate response - 201
+
+
+014 Create Chronic Lung Disease (Infectious disease of lung - disorder)
```
Comment (wlad) [inline tests/robot/CONDITION/01_create.robot]: der kann dann weg.

**Review by wlad — CHANGES_REQUESTED** [review]
as agreed, TC 14 and related JSON can be removed

File: src/test/resources/Condition/create-chronic-lung-disease-infectious-disease-disorder.json (hunk 1 above)
Comment (af-dpe) [reply src/test/resources/Condition/create-chronic-lung-disease-infectious-disease-disorder.json]: TC and json is removed

**Review by wlad — APPROVED** [review]
looks good 👍

**Pull request merged**